### ISSUE TITLE
User Image: New Photo Entity

### ISSUE THREAD

**Issue by Tonterias** [Tonterias/spingularchatsql6#13]

**Comment by Tonterias** [comment]
A User should have another entity for his photo. Now it is mixed on the ChatUser, so we need to let the user access the ChatUser to change the photo, but that is a problem because they can mess with the relation to the JHIUser and cause a problem. Once separated the USER will not be able to Edit/Delete the ChatUser once it is created.

**Comment by cchunduri** [comment]
I don't think so we need to create another entity for store user profile image data. Because Jhipster User object itself contains a parameter called imgurl which can hold location to profile pic URL. All we need is a neat UI for uploading profile pic.

**Comment by Tonterias** [comment]
Ok, how would you do it? Would it be easier to create such IU?

I think the new Entity Photo #18  is a good approach because Jhipster will do all the programming for us, but if you think that your approach is easier / cleaner / faster. It is ok with many.

Thanks,
